config/serverless.oblt.yml

```diff
@@ -34,6 +34,10 @@ xpack.observability.createO11yGenericFeatureId: true
 ## APM Serverless Onboarding flow
 xpack.apm.serverlessOnboarding: true
 
+# Synthetics mTLS cert locations
+xpack.uptime.service.tls.certificate: /mnt/elastic-internal/http-certs/tls.crt
+xpack.uptime.service.tls.key: /mnt/elastic-internal/http-certs/tls.key
```

Comment:
@pgayvallet i am kinda surprise that this is working, so in base serverless config we have 
`xpack.uptime.enabled: false`

even though this PR isn't removing that. it works and have enabled that plugin. Not sure how.